### DISCUSSION TITLE
How to remove the header in select component?

### DISCUSSION THREAD

**Discussion by DavidMaximaWang** [UI5/webcomponents#7495] [Q&A]
I have a ui5-select component.
In mobile device, even with very few dropdown items, it takes the whole height of the screen. 
But this is not I wanted. 
Is there a way to remove the header 
![image](https://github.com/SAP/ui5-webcomponents/assets/14916029/a2806085-b27b-4c43-904b-28dadf7d9efe)

**Comment by DavidMaximaWang** [comment]
My component is like:
`(<Select className={classes.modelSelect}>
         {[1,2].map(()=>(<Option>a</Option>))}
     </Select>
     )`

**Comment by petyabegovska** [comment]
Hi, 

Where the second header comes from? And which version of UI5 Web Components are you using?

Best regards,
Petya
